kolibri/plugins/learn/assets/src/vue/side-nav/nav-item.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+
+  <div class='link-wrapper'>
+    <a class='link' v-link="vlink" @click='closeSearch' :class="{active: active}">
```

Comment:
Having `closeSearch` here feels a bit weird/backwards -- what if there's other stuff that should happen when top-level nav is changed? Might make more sense to have the apps/plugins themselves listen for nav changes (some sort of "page exit" event?) and do whatever they want at that point.